analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -213,3 +216,17 @@ class Sbt(
         // This is not implemented in favor over overriding [resolveDependencies].
         throw NotImplementedError()
 }
+
+private fun moveGeneratedPom(pomFile: File): File {
+    val targetDirParent = pomFile.absoluteFile.parentFile.searchUpwardsForSubdirectory("target")
+        ?: return pomFile
```

Comment:
Nit: Just hinting, this would actually fit in one line.

Reply:
Done.